velocity/src/main/java/me/leoko/advancedban/velocity/command/CommandRecieverVelocity.java

```diff
@@ -0,0 +1,44 @@
+package me.leoko.advancedban.velocity.command;
+
+import com.velocitypowered.api.command.CommandSource;
+import com.velocitypowered.api.command.SimpleCommand;
+import com.velocitypowered.api.proxy.ProxyServer;
+import me.leoko.advancedban.Universal;
+import me.leoko.advancedban.manager.CommandManager;
+import java.util.Collections;
+import java.util.List;
+
+public class CommandRecieverVelocity implements SimpleCommand {
```

Comment:
```suggestion
public class CommandReceiverVelocity implements SimpleCommand {
```
Just noticed this was misspelled